nano/node/network.cpp

```diff
@@ -430,7 +430,8 @@ std::deque<std::shared_ptr<nano::transport::channel>> nano::network::list_non_pr
 // Simulating with sqrt_broadcast_simulate shows we only need to broadcast to sqrt(total_peers) random peers in order to successfully publish to everyone with high probability
```

Comment:
Is this comment out of date after this change?

'sqrt(total_peers)` should now be `log(total_peers)`, no?